docs/pages/cheatsheet.adoc

```diff
@@ -0,0 +1,732 @@
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+// Metadata
+:judo-reqprefix: REQ-INTR
+// Settings
+:idprefix:
+:idseparator: -
+:icons: font
+:KW: [purple]##**
+:KWE: **##
+:source-highlighter: highlight.js
+
+= Cheatsheet
+
+== Model
+
+No syntax change.
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**model**# &lt;name> [purple]#**;**#
+----
+
+*Example:*
+
+[source,jsl]
+----
+model shop;
+----
+
+[source,jsl]
+----
+model judo::example::shop;
+----
+
+== Import
+
+No syntax change.
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**import**# &lt;model> &#x5B;[purple]##**as**## &lt;alias>] [purple]#**;**#
+----
+
+*Example:*
+
+[source,jsl]
+----
+import judo::types;
+----
+
+If no alias is specified, the elements of the imported model can be accessed by their simple name or by their fully qualified name.
+
+[source,jsl]
+----
+import judo::types as types;
+----
+
+If an alias is specified, the elements of the imported model can be accessed by their name with the alias prefix or by their fully qualified name.
+
+== Type
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**type**# &lt;basetype> &lt;name> &#x5B;&lt;parameter name> [purple]#**:**# &lt;parameter value>] ... [purple]#**;**#
+----
+
+*Old syntax:*
+
+[subs="quotes"]
+----
+[purple]#**type**# &lt;basetype> &lt;name> &#x5B; [purple]#**(**#&lt;parameter name> [purple]#**=**# &lt;parameter value>][purple]#**,**# ...[purple]#**)**# ] [purple]#**;**#
+----
+
+*Example:*
+
+[source,jsl]
+----
+type boolean Boolean;
+type date Date;
+type time Time;
+type timestamp Timestamp;
+type numeric Integer precision:9 scale:0;
+type numeric Long precision:15 scale:0;
+type string String min-size:0 max-size:4000;
+----
+
+== Entity
+
+No syntax change.
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**entity**# &#x5B;[purple]##**abstract**##] &lt;name> &#x5B;[purple]##**extends**## &lt;entity>[purple]##**,**## ...] &#x5B; [purple]##**{**##
+    [member] ...
+[purple]#**}**# ] [purple]#**;**#
+----
```

Comment:
I think it's worth mentioning that from now on, it's not mandatory for the entity to have a body.

Reply:
If it's true, +1